distribution/docs/src/main/resources/content/_managing/attributes-added-by-pre-ingest-plugins.adoc

```diff
@@ -0,0 +1,12 @@
+:title: Attributes Added by Pre-Ingest Plugins
+:type: dataManagement
+:status: published
+:parent: Automatically Added Metacard Attributes
+:summary: How pre-ingest plugins add attributes to metacards.
+:order: 051
+
+* The <<_default_security_attribute_plugin,Default Security Attribute Plugin>> is a configurable plugin that maps system high user attribute values to metacard attribute values.
+"System high user attributes" refers to the attributes on the ${branding}'s https://en.wikipedia.org/wiki/Second-level_domain[Second-Level Domain] user, who is defined in the `etc/users.attributes` file.
```

Comment:
❓ This isn't quite correct. On installation, the user is in the `users.attributes` file but in a hardened system, that file is removed and an external attribute store should be queried. I think we should indicate that here.

@stustison, what is your thought on that?

Reply:
Without looking into it more, I think the only store that we pull the "System High Attributes" from is that text file. I'm not sure what happens when it is deleted.

Reply:
I agree with Richard about the hardened system. On doing a few minutes of investigation, it seems the Default Security Attributes Plugin pulls "system high user attributes" from the `UsersAttributesFileClaimsHandler` which requires the `users.attributes` file. Is that really the case?

Reply:
So after more investigation, I figured out that the `users.attributes` file never gets removed. It's the `users.properties` file that gets removed. `users.attributes` does not fully define the FQDN user. It just adds in attributes for that user. Which is also what's used for the "System high user attributes". That should be clarified here.

Reply:
Thanks for looking into this! I'll make sure to note that the users.attributes files just defines the attributes and not the entire user.